README.md

```diff
@@ -54,6 +54,7 @@ psql -Upostgres    # or "psql postgres", depending on your Pg installation
 # postgres=# grant all privileges on database openresty_org to openresty;
 
 make initdb   # create the database and load the data files
+make gentemp  # generate additional template files
```

Comment:
In the computing world, `temp` usually means "temporary" instead of "templates". So don't use this abbreviation for "templates" here. I don't like this extra step. Seems like this Makefile target should be part of the dependency chain of the existing `templates` target. After all, the default `make all` triggers `$(templates_lua)` generation. And your newly added `.tt2` files to be generated should really be listed in the existing `$(tt2_files)` variable value. In other words, we do not need this `gentemp` phony target in the first place.

Reply:
removed.